BUILDING.md

```diff
@@ -253,6 +253,23 @@ To install this version of Node.js into a system directory:
 $ [sudo] make install
 ```
 
+#### Building a debug build
+
+If you run into a issue where Node.js segfaults and the "segfault-handler" does not provide enough information, you might have to build a debug build of Node.js to get enough information to debug the issue further, this is done by :
```

Comment:
Nit: We generally wrap lines at 80 characters (and there’s an extra space before the `:`)

This can also be taken care of when landing, I’d say.